src/features/delegation/utils.ts

```diff
@@ -1,18 +1,21 @@
 import { accountsABI } from '@celo/abis';
 import { App, Octokit } from 'octokit';
 import path from 'path';
-import { fornoRpcUrl } from 'src/config/config';
+import { config, fornoRpcUrl } from 'src/config/config';
 import { Addresses } from 'src/config/contracts';
 import {
   delegateeRegistrationRequestToMetadata,
   metadataToJSONString,
 } from 'src/features/delegation/delegateeMetadata';
 import { RegisterDelegateRequest } from 'src/features/delegation/types';
 import { createPublicClient, http } from 'viem';
-import { celo } from 'viem/chains';
+import { celo, celoAlfajores } from 'viem/chains';
 
 export async function isAddressAnAccount(address: HexString) {
-  const client = createPublicClient({ chain: celo, transport: http(fornoRpcUrl) });
+  const client = createPublicClient({
```

Comment:
this is actually called in many places, so imho worth DRYing it, same for `createWalletClient`